controllers/routing/exported_svc_locator.go

```diff
@@ -0,0 +1,62 @@
+package routing
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/opendatahub-io/odh-platform/pkg/metadata"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/util/retry"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func getExportedServices(ctx context.Context, cli client.Client, target *unstructured.Unstructured) ([]corev1.Service, error) {
+	ownerName := target.GetName()
+	ownerKind := target.GetObjectKind().GroupVersionKind().Kind
+
+	listOpts := []client.ListOption{
+		client.InNamespace(target.GetNamespace()),
+		// TODO(mvp): centralize label creation
+		client.MatchingLabels{
+			metadata.Labels.RoutingExported: "true",
+			metadata.Labels.OwnerName:       ownerName,
+			metadata.Labels.OwnerKind:       ownerKind,
+		},
+	}
```

Comment:
worth adding UID label matching here? 

follow up question (maybe sth for future iteration past MVP) - not sure if `client.InNamespace(target.GetNamespace()),` will work for cluster scoped resources? worth checking at some pt.

Reply:
> worth adding UID label matching here?

We don't need to, UID is our internal metadata and SVC is namespace-scoped there is no risk in duplication here.

> follow up question (maybe sth for future iteration past MVP) - not sure if client.InNamespace(target.GetNamespace()), will work for cluster scoped resources? worth checking at some pt.

In general, it won't work, but `Service` (and `VirtualService` later too) which we are exporting are namespaced-scope resources, so all good for this case.